Sources/Commands/SwiftTool.swift

```diff
@@ -433,22 +433,33 @@ public class SwiftTool<Options: ToolOptions> {
         guard result.exitStatus == .terminated(code: 0) else {
             throw ProcessResult.Error.nonZeroExit(result)
         }
+
+        // Create backwards-compatibilty symlink to old build path.
+        let oldBuildPath = buildPath.appending(component: options.configuration.dirname)
+        if exists(oldBuildPath) {
+            try removeFileTree(oldBuildPath)
+        }
+        try createSymlink(oldBuildPath, pointingAt: plan.buildParameters.buildPath, relative: true)
```

Comment:
Yes I did. We're not relying on it for tests to pass, but somebody might depend on it unwittingly in the future. I wonder if we could make into an API-level option that we disable by default for tests. What do you think? Probably a future PR.